src/main/scala/mesosphere/marathon/core/appinfo/impl/DefaultInfoService.scala

```diff
@@ -20,7 +20,7 @@ private[appinfo] class DefaultInfoService(
     newBaseData: () => AppInfoBaseData)(implicit ec: ExecutionContext)
   extends AppInfoService with GroupInfoService with PodStatusService with StrictLogging {
 
-  override def selectPodStatus(id: PathId, selector: PodSelector): Future[Option[PodStatus]] =
+  override def selectPodStatus(id: PodDefinition.PodKey, selector: PodSelector): Future[Option[PodStatus]] =
```

Comment:
We use `AbsolutePathId` everywhere else - why do we need `type PodKey` alias here?

Reply:
Looks like PodKey, GroupKey, and AppKey are just aliased; I'm not sure I see a lot of value in adding the alias.

Reply:
Yeah, I think there was an AppKey and GroupKey already, and I halfheartedly added the PodKey. I kind of like the idea of having an alias for Keys, but it's pretty hard to make this really consistent everywhere. 
I think i'll just remove the aliases and use AbsolutePathId everywhere.